index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="UTF-8">
+    <title>PortfolioS</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <header class="container">
```

Comment:
- [Optional] Great job using a header element. I noticed that the spaces to the left and right of your header is more than that on the template. You are expected to stick to design guidelines as much as possible and this includes margins and paddings. Kindly fix this please.
- Figma
![Screenshot_2023-04-28-15-01-49-466_com android chrome~2](https://user-images.githubusercontent.com/94551484/235169576-73d9d537-7a10-4e2b-8784-6a3685e5e041.jpg)
- Your design 
![Screenshot_2023-04-28-14-56-57-442_com android chrome~2](https://user-images.githubusercontent.com/94551484/235169585-03bf47fb-488f-473e-a0a6-3aa0895fc345.jpg)